module/castArray.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @module 1-liners/castArray
+ *
+ * @description
+ *
+ * If the provided `value` is an array returns a copy of it otherwise returns an array containing the original `value`.
+ *
+ * @example
+ *
+ * 	const castArray = require('1-liners/castArray');
+ *
+ * 	castArray([1, 2, 3]);  // => [1, 2, 3]
+ * 	castArray(1);  				 // => [1]
+ * 	castArray(null);  	   // => [null]
+ *
+ */
+export default (value) => Array.isArray(value) ? value.slice(0) : [value];
```

Comment:
@stoeffel, I've read that `.slice(0)` is the fastest. `[...value]` is currently transpiled to a weird thing, see https://babeljs.io/repl/#?evaluate=true&presets=es2015&experimental=true&loose=false&spec=true&code=%5B...value%5D.

On the other hand it is probably more readable.